tests/meltano/core/test_project_add_service.py

```diff
@@ -103,8 +103,8 @@ def test_add_inherited(
             "inherit_from": "tap-mock-inherited",
         }
 
```

Comment:
@aaronsteers @edgarrmondragon -- I commented all of the other changed tests to explain why lockfiles were being removed there.

The diffs in this file are because if I'm understanding correctly, now meltano will only reach out to the hub _once_ and then use the plugin definition it finds in the lockfile. So these should all be `1` now

Reply:
@cjohnhanson that's correct. Once the lockfile has been added, there should only be 1 trip to Hub for each endpoint.